core/template.go

```diff
@@ -11,7 +11,16 @@ var DisableColor = false
 
 var (
 	HelpInputRune = '?'
-	HelpIcon      = "ⓘ"
+
+	ErrorIcon    = "✘"
+	HelpIcon     = "ⓘ"
+	QuestionIcon = "?"
+
+	MarkedOptionIcon   = "◉"
+	UnmarkedOptionIcon = "◯"
+
+	MultiSelectFocusIcon = "❯"
```

Comment:
Yeah, I was thinking that, consistency would probably be a good idea.  Which one wins?